code/game/objects/items/stacks/medical.dm

```diff
@@ -285,6 +284,57 @@
 		return
 	. = ..()
 
+/obj/item/stack/medical/mesh/advanced
+	name = "advanced regenerative mesh"
+	desc = "An advanced mesh made with aloe extracts and sterilizing chemicals, used to treat burns."
+
+	gender = PLURAL
+	singular_name = "advanced regenerative mesh"
+	icon_state = "aloe_mesh"
+	heal_burn = 15
+	grind_results = list(/datum/reagent/medicine/spaceacillin = 1)

```

Comment:
Grind result doesn't really make sense, considering what is required to mix this.